usr/src/uts/common/vm/vm_as.c

```diff
@@ -665,6 +666,7 @@ as_alloc(void)
 	as->a_lastgap		= NULL;
 	as->a_lastgaphl		= NULL;
 	as->a_callbacks		= NULL;
+	as->a_proc		= NULL;
```

Comment:
@rmustacc I added this line per your prior comment. Please let me know if this isn't what you had in mind.